src/popup/pages/Paratime/index.js

```diff
@@ -93,29 +101,27 @@ class Paratime extends React.Component {
         const { currentAccount } = this.props
         let rpcBalance = getRpcBalance(currentAccount.address)
         let runtimeRequestList = getRpcRuntimeList()
-        Promise.all([rpcBalance, runtimeRequestList]).then(async (data) => {
-            let rpcAccount = data[0]
+        try {
+            const [rpcAccount, runtimeList] = await Promise.all([rpcBalance, runtimeRequestList])
             let allowanceList = rpcAccount.allowanceList
-
-            let runtimeList = data[1]
             let newRuntimeList = await this.getRuntimeDetail(runtimeList, allowanceList)
             this.props.updateRuntimeList(newRuntimeList)
-            this.callSetState({
-                refreshing: false,
-                loading: false,
-                currentShowList: this.getShowRuntimeList()
-            }, () => {
-                this.isRequest = false
-                this.notifyAccountLoading()
-            })
-        }).catch((error) => {
+        } catch (error) {
             this.callSetState({
                 refreshing: false,
                 loading: false
             }, () => {
                 this.isRequest = false
                 this.notifyAccountLoading()
             })
+        }
+        this.callSetState({
+            refreshing: false,
+            loading: false,
```

Comment:
Did you mean to remove the duplicate thing in `} catch (error) {`?

This seems best wrapped into `finally`
```ts
try {
  ...
  this.props.updateRuntimeList(newRuntimeList)
} finally {
  this.callSetState({ 
    refreshing: false
  ... 
}
```

Reply:
No, this is a different statement moved from the `then` above. I see that there are some lines in common though :thinking: 

But to keep this as low risk as possible, I'm keeping it this way in this PR.